Sources/MessagingPush/PushHandling/NotificationCenterDelegate.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import UserNotifications
+
+protocol PushEventListener {
```

Comment:
This protocol is our SDK's equivalent to `UNUserNotificationCenterDelegate`. By having our own version of `UNUserNotificationCenterDelegate` with functions and parameters that our tests can call, this can make our SDK's `UNUserNotificationCenterDelegate` functions testable.